### ISSUE TITLE
Start WebUI and install WebAPI

### ISSUE THREAD

**Issue by niXta1** [binhex/arch-deluge#8]
Hi!

I'm trying to install the [WebAPI-plugin](https://github.com/idlesign/deluge-webapi) but I think I need to start the WebUI first acording to [this discussion](https://github.com/idlesign/deluge-webapi/issues/27).

I can't see anything suspicious in the logs when I try to start WebUI, it just won't check the box.

**Comment by binhex** [comment]
see Q3 and Q4:-
https://github.com/binhex/documentation/blob/master/docker/faq/delugevpn.md

**Comment by niXta1** [comment]
> see Q3 and Q4:-
> https://github.com/binhex/documentation/blob/master/docker/faq/delugevpn.md

Right, I thought something was wrong because it didn't respond like the other plugins. After enabling in core.conf it stayed checked. I still get the exact same error in the log when I enable WebAPI.
